gr-utils/python/modtool/modtool_add.py

```diff
@@ -73,17 +73,21 @@ def setup(self, options):
 
         self._info['blocktype'] = options.block_type
         if self._info['blocktype'] is None:
-            # Print(list out of blocktypes to user for reference)
+            # Print list out of blocktypes to user for reference
             print(str(self._block_types))
-            while self._info['blocktype'] not in self._block_types:
-                self._info['blocktype'] = input("Enter block type: ")
-                if self._info['blocktype'] not in self._block_types:
-                    print('Must be one of ' + str(self._block_types))
+            with SequenceCompleter(sorted(self._block_types)):
+                while self._info['blocktype'] not in self._block_types:
+                    self._info['blocktype'] = input("Enter block type: ")
+                    if self._info['blocktype'] not in self._block_types:
+                        print('Must be one of ' + str(self._block_types))
+
         # Allow user to specify language interactively if not set
         self._info['lang'] = options.lang
         if self._info['lang'] is None:
-            while self._info['lang'] not in ['c++', 'cpp', 'python']:
-                self._info['lang'] = input("Language (python/cpp): ")
+            language_candidates = ('cpp', 'python')
```

Comment:
did I or did you omit `'c++'`? If I did it, I can't remember the rationale. If you did it: why is there no `c++`?